prober/prober.go

```diff
@@ -288,6 +293,22 @@ func (p *Probe) loop() {
 		return
 	}
 
+	if p.IsContinuous() {
+		// Probe function is going to run continuously.
+		for {
+			p.run()
+			// Wait and then retry if probe fails. We use the inverse of the
+			// configured negative interval as our sleep period.
+			// TODO(percy):implement exponential backoff, possibly using logtail/backoff.
```

Comment:
probably worth importing the backoff package doing this now?

Reply:
alternatively we should document the negative interval translating to continuous wait-before-retry on `Probe.interval`

Reply:
`logtail.Backoff` may not actually be a great fit, as it seems to be better for repetitive operations of finite duration upon whose success we can reset the backoff. In our case, it's really just a single very long running operation, so we would never reset the backoff.

I've updated the comment on `Prober.Run()` since that's the exported function that callers will use.